<ui version="4.0" >
 <class>preferencesDialog</class>
 <widget class="QDialog" name="preferencesDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>285</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Préférences</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>206</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <item>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="templatesPathLabel" >
        <property name="text" >
         <string>Chemin d'accès aux maquettes</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLineEdit" name="templatesPathEdit" />
      </item>
      <item row="1" column="1" >
       <widget class="QPushButton" name="templatesPathButton" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" >
      <item row="0" column="0" colspan="2" >
       <widget class="QLabel" name="inkscapePathLabel" >
        <property name="text" >
         <string>Chemin d'accès à Inkscape</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLineEdit" name="inkscapePathEdit" />
      </item>
      <item row="1" column="1" >
       <widget class="QPushButton" name="inkscapePathButton" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Compteur de transformations XSL</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="transformCounter" >
          <property name="text" >
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resetTransformCounterButton" >
          <property name="text" >
           <string>Réinitialiser</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>preferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>preferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
